docs/providers.rst

```diff
@@ -301,11 +301,11 @@ EthereumTesterProvider
 
 .. py:class:: EthereumTesterProvider(eth_tester=None)
 
-    This provider integrates with the ``eth-tester`` library.  The
-    ``eth_tester`` constructor argument should be an instance of the
-    :class:`~eth_tester.EthereumTester` class provided by the ``eth-tester``
-    library.  If you would like a custom eth-tester instance to test with,
-    see the ``eth-tester`` library documentation for details.
+    This provider integrates with the ``eth-tester`` library.  The ``eth_tester`` constructor
+    argument should be an instance of the :class:`~eth_tester.EthereumTester` or
+    :class:`~eth_tester.PyEVMBackend` class provided by the ``eth-tester`` library.
```

Comment:
I'm not sure if encouraging the use of `PyEVMBackend` directly is a good idea or not in the docs. Is it better to just promote that `eth_tester.EthereumTester` is the only way to instantiate an `EthereumTesterProvider` instance, and the code will be silently forgiving to anybody who mistakenly provides an unwrapped `PyEVMBackend`?